packages/ra-input-rich-text/src/index.tsx

```diff
@@ -123,20 +157,13 @@ const RichTextInput = ({
     );
 };
 
-RichTextInput.propTypes = {
-    label: PropTypes.string,
-    options: PropTypes.object,
-    source: PropTypes.string,
-    toolbar: PropTypes.oneOfType([
-        PropTypes.array,
-        PropTypes.bool,
-        PropTypes.shape({
-            container: PropTypes.array,
-            handlers: PropTypes.object,
-        }),
-    ]),
-    fullWidth: PropTypes.bool,
-    configureQuill: PropTypes.func,
+RichTextInput.defaultProps = {
```

Comment:
Please keep these as default values for the props instead, React is deprecating `defaultProps` for function components (https://github.com/facebook/react/pull/16210).

Reply:
:+1: